src/llama_stack_client/lib/inference/event_logger.py

```diff
@@ -28,6 +28,6 @@ def log(self, event_generator):
             if event.event_type == "start":
                 yield LogEvent("Assistant> ", color="cyan", end="")
             elif event.event_type == "progress":
-                yield LogEvent(event.delta, color="yellow", end="")
+                yield LogEvent(event.delta.text, color="yellow", end="")
```

Comment:
this is not correct. you should check the type of the delta and then if it is a text delta print this or if there is a tool call in there, print that